README.md

```diff
@@ -627,6 +627,35 @@ Compile [tx-fuzz](https://github.com/MariusVanDerWijden/tx-fuzz) locally per ins
 ./livefuzzer spam  --sk "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" --rpc http://127.0.0.1:<port> --slot-time 2
 ```
 
+#### Run contender to send l2 transactions
+
+Install the latest [contender](https://github.com/flashbots/contender) version via cargo:
+```bash
+cargo install --git https://github.com/flashbots/contender --bin contender --force 
+```
+
+Browse the available [scenarios](https://github.com/flashbots/contender/tree/main/scenarios) and pick one that fits your needs. For example, to download the `stress` scenario:
+```bash
+wget https://raw.githubusercontent.com/flashbots/contender/refs/heads/main/scenarios/stress.toml -O scenario.toml
+```
+
+In the snippet below, we use Docker to inspect the container named `op-el-1` and dynamically extract the port. You can replace this approach with a specific L2 RPC port if you prefer::
```

Comment:
Thoughts on making it easy to run this from within kurtosis itself?

Reply:
We could run this as component, but the problem is that after some time funds on the funded address are used up and you need to use different address/top it up somehow.
Also we'll need an easy to way to provide custom toml scenarios for contender.
So, while it's doable i think easy way is to use external contender.

Reply:
We could leave this as-is and add this use-case to the new devnet SDK testing framework goals @sigma